nyxx/lib/src/core/channel/guild/GuildChannel.dart

```diff
@@ -132,3 +104,37 @@ abstract class GuildChannel extends IChannel {
   Future<Invite> createInvite({int? maxAge, int? maxUses, bool? temporary, bool? unique, String? auditReason}) =>
       client._httpEndpoints.createInvite(this.id, maxAge: maxAge, maxUses: maxUses, temporary: temporary, unique: unique, auditReason: auditReason);
 }
+
+abstract class MinimalGuildChannel extends IChannel {
+  /// The channel"s name.
```

Comment:
```suggestion
  /// The channel's name.
```
Thats my mistake but pls fix